### PR TITLE
Adds TNG multiband images as dataset

### PR DESCRIPTION
**Pull request by EiffL**

### CODE REVIEW

File: sfh/datasets/mergers/kinetic_reduced.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+"""Mergers_kinetic reduced dataset."""
```
Comment (EiffL) [inline sfh/datasets/mergers/kinetic_reduced.py:1]: This is a good idea, but there is a actually a better way to do this ;-) you can define variants of TensorFlow datasets, here is one example from a package I develop:
https://github.com/CosmoStat/autometacal/blob/974b2711ca9fd840c551fab30b0775ab2e5b0876/autometacal/python/datasets/gal_gen.py#L49

**Review by EiffL — COMMENTED** [review]
This is looking really good! Only made one small comment

File: sfh/datasets/mergers/kinetic_reduced.py (hunk 1 above)
Comment (EiffL) [reply sfh/datasets/mergers/kinetic_reduced.py:1]: but it's more to let you know that this is possible than anything else.

Comment (ppfn) [reply sfh/datasets/mergers/kinetic_reduced.py:1]: Thanks @EiffL, I'll have a look at it and try to modify our dataset accordingly.

Comment (ppfn) [reply sfh/datasets/mergers/kinetic_reduced.py]: Commit a9f30bddac1b8fc89f877902b24ed151df3bb3e6 implement variant of the dataset MergersKinetic.

**Review by EiffL — COMMENTED** [review]
LGTM

**Pull request merged**